.travis.yml

```diff
@@ -3,6 +3,8 @@
 
 language: csharp
 mono: 6.4.0
+os: linux
```

Comment:
Travis's validator gives warnings about these being undefined, even though it doesn't seem to matter.